common/step_setbootorder.go

```diff
@@ -0,0 +1,41 @@
+package common
+
+import (
+	"context"
+	"github.com/hashicorp/packer/helper/multistep"
+	"github.com/hashicorp/packer/packer"
+	"github.com/jetbrains-infra/packer-builder-vsphere/driver"
+	"strings"
+)
+
+type StepSetBootOrder struct {
+	BootOrder string
+	SetOrder bool
+}
+
+func (s *StepSetBootOrder) Run(_ context.Context, state multistep.StateBag) multistep.StepAction {
+	ui := state.Get("ui").(packer.Ui)
+	vm := state.Get("vm").(*driver.VirtualMachine)
+
+	if s.BootOrder != "" {
+		ui.Say("Set boot order...")
+		order := strings.Split(s.BootOrder, ",")
+		if err := vm.SetBootOrder(order); err != nil {
+			state.Put("error", err)
+			return multistep.ActionHalt
+		}
+	} else {
```

Comment:
Since you're anyway editing this code, please convert into `else if s.SetOrder` :)

Reply:
Done! :)